levelkdev/swapr-dapp
# WIP: Add liquidity list sort by

### WIP 
Me and @Diogomartf are working on updating the design of liquidity page and also the fonts and colors._

This is **work in progress** it's not finished, but we want to divide the work in smaller chunks and do things incrementally. 

We will do this smaller PRs and merge it on a "main" branch, `feature/fonts-and-colors-liquidity-page`.

--- 

# Summary

Closes #891

Add "sort by" dropdown and sort list by chosen option

<img width="1423" alt="Screenshot 2022-04-21 at 14 10 43" src="https://user-images.githubusercontent.com/23079677/164465152-db3c3f65-a911-4ac9-a6fd-0fbd64b72351.png">

  # To Test

1. Open the page `pools`
2. Choose new sort by option
- [ ] Verify if list order corresponds to the chosen option
